Sources/Valkey/Connection/ValkeyConnection.swift

```diff
@@ -144,8 +144,10 @@ public final actor ValkeyConnection: ValkeyClientProtocol, Sendable {
         self.channel.close(mode: .all, promise: nil)
     }
 
-    func waitOnActive() async throws {
+    func initialHandshake() async throws {
         try await self.channelHandler.waitOnActive().get()
+        self.executeAndForget(command: CLIENT.SETINFO(attr: .libname("valkey-swift")))
+        self.executeAndForget(command: CLIENT.SETINFO(attr: .libver("0.1.0")))
```

Comment:
we should get this string from a static version string, that we maintain in `Version.swift` or whatever. Before every release we should bump it.